src/flags.zig

```diff
@@ -72,7 +72,34 @@ pub fn fatal(comptime fmt_string: []const u8, args: anytype) noreturn {
 pub fn parse_commands(args: *std.process.ArgIterator, comptime Commands: type) Commands {
     assert(@typeInfo(Commands) == .Union);
 
-    const first_arg = args.next() orelse fatal("subcommand required", .{});
+    const first_arg = args.next() orelse {
+        const fields = comptime blk: {
+            var fields: [:0]const u8 = "";
+            inline for (std.meta.fields(Commands)) |field, i| {
```

Comment:
I _think_ this could be just `for`, without `inline`, because we are already in comptime.

Reply:
You're right! Dropped.